### ISSUE TITLE
Find - Hide Disabled Events

### ISSUE THREAD

**Issue by Jase0000** [Scirra/Construct-feature-requests#101]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

The ability to hide "disabled events" in find results would be extremely useful!

### Possible workarounds or alternatives

I guess not having many disabled events would be a workaround.

A possible alternative would be to "dim the colour" of disabled events in the find results, just so it's visually quicker to scroll through and see what's lit up and what is darkened. 

### Proposed solution

A small tick box to hide "disabled events" in find results. 

Hopefully a minor change (not claiming it is, but I have hope!!). 

### Why is this idea important?

Seems trivial but I've identified this is something costing me time scrolling around on my large project. 

I tend to disable events that I wish to keep, or old systems or old attempts, and these have indeed racked up and pollute the search results. I am glad I kept these disabled as I sometimes revert back to an old system, or use some pieces of a disabled system in a new system, major time saver! 

I also like to add actions everywhere for a "debug SpriteFont" to output information often. Whilst I understand the browser console could be used for this, I like to have debug info directly in front of me, sometimes catching things I may not have spotted unless I chose to open the browser console.

Due to this, I have many disabled events for these SpriteFonts, I like to keep them especially as it's quick to "Enable, test again" if something mysteriously breaks.

But currently I'm scrolling through massive lists to find a a SpriteFont action that I wish to disable. Very tricky to spot since I'm looking for the "result that isn't striked through". To be able to hide these would shrink hundreds of results to mere 10 results or so, saving so much time.

**Comment by WilsonPercival** [comment]
Yes, that would help me. I often decompose other people's projects, which contain thousands of events. Sometimes I use search to find the action I need and see a lot of disabled lines in front of me.
